docs/chart/installation-with-helm.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Please merge this with other Markdown files into one README.md that will be placed in `chart/githubconnector/` directory.